cockroachcloud/quickstart.md

```diff
@@ -29,29 +31,22 @@ To run CockroachDB on your local machine instead, see [Start a Local Cluster](..
 
     Your cluster will be created in a few seconds and the **Connection info** dialog will display.
 
-1. Click the **Connection string** tab in the **Connection info** dialog.
-    1. Run the command in step 1 to download the CA certificate to your local machine.
-    1. Copy the connection string in step 2 to a secure location.
-
-        {{site.data.alerts.callout_danger}}
-        The connection string in the command is pre-populated with your username, password, cluster name, and other details. Your password, in particular, will be provided only once. Save it in a secure place (Cockroach Labs recommends a password manager) to connect to your cluster in the future. If you forget your password, you can reset it by going to the [**SQL Users** page](user-authorization.html).
-        {{site.data.alerts.end}}
-
 ## Step 2. Connect to the cluster
 
-The **Connection info** dialog shows information about how to connect to your cluster for the client OS.
+The **Connection info** dialog shows information about how to connect to your cluster.
 
-<div class="filters clearfix">
-  <button class="filter-button page-level" data-scope="mac">Mac</button>
-  <button class="filter-button page-level" data-scope="linux">Linux</button>
-  <button class="filter-button page-level" data-scope="windows">Windows</button>
-</div>
+1. Click the **Choose your OS** dropdown, and select the operating system of your local machine.
+1. Click the **Connection string** tab in the **Connection info** dialog.
+    1. Open a new terminal on your local machine, and run the command provided in step **1** to download the CA certificate. This certificate is required by most clients connecting to {{ site.data.products.db }}.
```

Comment:
I'd prefer we only instruct the user to download a certificate if they want to use a programming language that requires it. I guess that would imply splitting the quickstart by language.